guides/tutorials/codelabs/azure-vmss-source-to-prod/index.md

```diff
@@ -0,0 +1,326 @@
+
+```bash
+ssh -L 8080:localhost:8080 -L 9000:localhost:9000 -L 8084:localhost:8084 -L 8087:localhost:8087 username@vmdnsprefix.selected_region.cloudapp.azure.com
+```
+
+> NOTE: You will have to enter the password that you have specified in the previous step.
+
+
+## 1. Configure Jenkins 
+
+1. After you have started your ssh tunnel, navigate to `http://localhost:8080/` on your local machine.
+1. The instance should already be unlocked and your first account setup. Login with the credentials you specified when deploying the template.
+1. Click **Install suggested plugins**: 
+   ![Install Jenkins plugins](images/1-Install-Jenkins-plugins.png)
+1. Your Jenkins instance is now ready to use!
+1. Run the `Sample Aptly Job` by clicking on the run button for the job. Verify that the job completes successfully.
+
+   ![Run Jenkins job](images/1-run-sample-aplty-job.png)
+
+## 2. Configure Spinnaker
+
+### I. Create the "codelab" application 
+
+1. Navigate to your Spinnaker VM by using `http://localhost:9000` on your local machine. 
+1. Click on the **Actions** menu on the right
+
+   ![Actions](images/2-actions-create-applications.png)
+   
+1. In the *New Application* window, enter the following values: 
+  * Name: codelab
+  * Owner Email: Enter your email
+  
+    And click **Create**
+  
+    ![New Application](images/2-create-application.png)
+  
+    You can read more how the Spinnaker application matches to Azure resources in the [Azure reference for Spinnaker](/reference/providers/azure.md)
+
+### II. Create a Security Group"
+
+1. Navigate to the [SECURITY GROUPS](http://localhost:9000/#/applications/codelab/securityGroups) page and click on the **Create Security Group** button.
+1. Provide the following values in the *Create New Security Group* dialog: 
+  * Detail: web
+  * Account: my-azure-account (should be selected by default)
+  * Region: westus (select the region that is closest to your location)
+
+    Ingress (click *Add new Security Group Rule*):
+  * Protocol: TCP
+  * Start Port: 80
+  * End Port: 80 
+
+    And click **Create**, if the loadbalancer does not appear in the page once the creation is complete, refresh your browser.
+
+    ![New Security Group](images/2-create-security-group.png)
+
+  > Note: We are using westus for the region but you can select any other region that is closer to your location. Be careful to match the region in the following steps.
+
+  You can read more about the implementation of Security Groups in Azure in the [Azure reference for Spinnaker](/reference/providers/azure.md)
+
+### III. Create a Load Balancer
+
+1. Navigate to the [LOAD BALANCERS](http://localhost:9000/#/applications/codelab/loadBalancers) page and click the **Create Load Balancer** button.
+1. Provide the following values in the "Create New Load Balancer" dialog: 
+  * Account: my-azure-account
+  * Region: westus (unless you have selected a different region previously)
+  * Virtual Network: devopsVnet

```

Comment:
Maybe add a note saying "this VNET was created for you as a part of the quickstart template"